readthedocs/core/mixins.py

```diff
@@ -34,41 +28,36 @@ class ProxiedAPIMixin:
 class CDNCacheControlMixin:
 
     """
-    Allow to cache views at the CDN level when privacy levels are enabled.
+    Explicitly cache or not a view at the CDN level.
 
-    The cache control header is only used when privacy levels
-    are enabled (otherwise everything is public by default).
+    The cache control header is used to mark the response as public/private,
+    so it can be cached or not.
 
     Views that can be cached should always return the same response for all
     users (anonymous and authenticated users), like when the version attached
     to the request is public.
 
-    To cache a view you can either set the `cache_request` attribute to `True`,
+    To explicitly cache a view you can either set the `cache_response` attribute to `True`/`False`,
     or override the `can_be_cached` method.
```

Comment:
Which takes precedence? We should likely document the order we check things, so it's clear how the logic is interpreted.

Reply:
Why do we have two different ways of doing the same? Can we have just one? 😄